src/api/users.ts

```diff
@@ -0,0 +1,6 @@
+import { User } from '../types/User';
+import { client } from '../utils/fetchClient';
+
+export const getUsers = () => {
+  return client.get<User[]>(`/users`);
```

Comment:
Consider adding error handling for the `getUsers` API call. Including a `.catch` block will help manage potential errors during the request and provide feedback to the user if the user retrieval fails.